index.css

```diff
@@ -8,55 +8,105 @@
   }
 }
 
-@layer components {
-  .calendar {
-    @apply border-l-neutral-950
-  }
-  .border {
-    @apply border-black border-2
-  }
-  .btn {
-    @apply bg-green-700  py-3 px-6 font-serif text-sm;
-  }
-  .button {
-    @apply !text-primary-textColor !bg-primary-pageColor h-10 w-2 
-  }
-  .button:disabled {
-    @apply !bg-primary-blurColor !text-transparent
-  }
-  .button:enabled:hover,
-  .button:enabled:focus {
-    @apply !bg-primary-secondaryTabColorBorderColor !border-l-zinc-950
-  }
-  .react-calendar {
-    @apply  w-fit !border-l-zinc-950
-  }
-  .react-calendar__navigation__arrow {
-    @apply hidden
-  }
- 
+.react-calendar {
+  @apply bg-white;
+  border-spacing: 0;
+  border-collapse: collapse;
+  @apply rounded-none !important;
+  @apply w-[262.5px] bg-white !important;
+  padding: 5px;
+  border: 1px solid #ccc;
+}
 
-  .react-calendar__month-view {
-    @apply grid grid-flow-col gap-0
-  }
-  .react-calendar__tile--current {
-    @apply pl-2;
-  }
-  .react-calendar__month-view__days__day--neighboringMonth {
-    @apply bg-gray-50 text-black; 
-  }
-  .react-calendar__month-view__weekNumbers .react-calendar__tile {
-    @apply flex align-middle justify-center font-extrabold border-black border-2
-  }
+.react-calendar__month-view__days {
+  @apply gap-0;
+}
 
-  .react-calendar__tile {
-    @apply  !border-black border-2
-  }
-  .react-calendar__month-view__weekdays {
-    @apply align-middle font-bold border-primary-textColor !no-underline text-primary-primaryTabColor text-lg
-  }
-  .react-calendar__navigation {
-    @apply flex justify-start text-xl align-top !border-l-zinc-950 text-primary-secondaryTabColor bg-primary-primaryTabColor font-bold
+.react-calendar__month-view__days__day {
+  @apply p-0 m-0 !important;
+  flex: none !important;
+}
 
-  }
+.react-calendar__month-view__days__day--neighboringMonth {
+  @apply text-gray-400 pointer-events-none;
+}
+
+.react-calendar__month-view__days__day--neighboringMonth > div {
+  @apply opacity-0;
+}
+
+.react-calendar__tile {
+  @apply p-0 m-0;
+  max-width: initial !important;
+}
+
+/* Remove hover, active and now states */
+.react-calendar__tile:hover,
+.react-calendar__tile--active,
+.react-calendar__tile--now {
+  @apply bg-white;
+}
+
+.react-calendar__month-view__weekdays {
+}
+
+.react-calendar__month-view__weekdays__weekday {
+  font-size: 11px !important;
+  color: #295da7;
+  padding: 0 !important;
+  width: 35.8px !important;
+  height: 20.4px !important;
+  flex: none !important;
+  text-transform: capitalize !important;
```

Comment:
@Harsh71019 , we won't transform the text anywhere. If needed in particular style then it should be coming in same format from data

Reply:
u mean that we pass css in json ?